apollo-runtime/src/appleMain/kotlin/com/apollographql/apollo3/network/http/NSURLSessionHttpEngine.kt

```diff
@@ -95,7 +95,6 @@ class NSURLSessionHttpEngine(
   }
 }
 
-@OptIn(ExperimentalUnsignedTypes::class)
```

Comment:
😎  TIL this was promoted to Stable in 1.5 🎉  https://kotlinlang.org/docs/whatsnew15.html#stable-unsigned-integer-types